content/id/docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -0,0 +1,1202 @@
+---
+title: Dasar-dasar StatefulSet
+content_type: tutorial
+weight: 10
+---
+
+<!-- overview -->
+Tutorial ini memberikan pengantar untuk manajemen aplikasi dengan
+{{< glossary_tooltip text="StatefulSet" term_id="statefulset" >}}.
+Di sini dicontohkan bagaimana cara untuk membuat, menghapus, melakukan penyekalaan, 
+dan memperbarui Pod dari StatefulSet.
+
+
+## {{% heading "prerequisites" %}}
+
+Sebelum memulai tutorial ini, kamu harus mengakrabkan dirimu dengan
+konsep-konsep Kubernetes sebagai berikut:
+
+* [Pod](/id/docs/concepts/workloads/pods/)
+* [DNS klaster](/id/docs/concepts/services-networking/dns-pod-service/)
+* [Service _headless_](/id/docs/concepts/services-networking/service/#service-headless)
+* [PersistentVolume](/id/docs/concepts/storage/persistent-volumes/)
+* [Penyediaan PersistentVolume](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/staging/persistent-volume-provisioning/)
+* [StatefulSet](/id/docs/concepts/workloads/controllers/statefulset/)
+* Alat baris perintah (_command line tool_) [kubectl](/docs/reference/kubectl/kubectl/)
+
+{{< note >}}
+Tutorial ini mengasumsikan bahwa klastermu telah dikonfigurasikan untuk 
+melakukan penyediaan PersistentVolume secara dinamis. Jika klastermu belum 
+dikonfigurasikan seperti itu, kamu harus menyediakan dua volume masing-masing dengan 
+ukuran 1 GiB sebelum memulai tutorial.
+{{< /note >}}
+
+## {{% heading "objectives" %}}
+
+StatefulSet ditujukan untuk digunakan dengan aplikasi-aplikasi _stateful_ 
+dan sistem terdistribusi. Akan tetapi, tata kelola aplikasi-aplikasi _stateful_ 
+dan sistem terdistribusi pada Kubernetes merupakan topik yang luas dan kompleks. 
+Untuk menunjukkan fitur-fitur dasar dari StatefulSet dan tidak mencampuradukkan 
+topik sebelum dan terakhir, kamu akan menggelar sebuah aplikasi web sederhana 
+menggunakan StatefulSet. 
+
+Setelah tutorial ini, kamu akan akrab hal-hal berikut:
+
+* Bagaimana cara membuat sebuah StatefulSet
+* Bagaimana suatu StatefulSet mengelola Pod
+* Bagaimana cara menghapus StatefulSet
+* Bagaimana cara melakukan penyekalaan terhadap suatu StatefulSet
+* Bagaimana cara memperbarui Pod dari StatefulSet
+
+<!-- lessoncontent -->
+
+## Membuat Sebuah StatefulSet
+
+Mulailah dengan membuat sebuah Statefulset dengan menggunakan contoh di bawah ini.
+Hal ini mirip dengan contoh yang ditunjukkan di dalam konsep 
+[StatefulSet](/id/docs/concepts/workloads/controllers/statefulset/).
+Contoh ini menciptakan sebuah 
```

Comment:
nit: "menciptakan" seems closer to "creation from nothing", how about using "membuat"?
```suggestion
Contoh ini membuat sebuah 
```